azurerm/internal/services/network/tests/virtual_hub_route_table_test.go

```diff
@@ -0,0 +1,250 @@
+  location = "%s"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "acctest-vnet-%d"
+  address_space       = ["10.5.0.0/16"]
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_network_security_group" "test" {
+  name                = "acctest-nsg-%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "acctest-subnet-%d"
+  resource_group_name  = azurerm_resource_group.test.name
+  virtual_network_name = azurerm_virtual_network.test.name
+  address_prefixes     = ["10.5.1.0/24"]
+}
+
+resource "azurerm_subnet_network_security_group_association" "test" {
+  subnet_id                 = azurerm_subnet.test.id
+  network_security_group_id = azurerm_network_security_group.test.id
+}
+
+resource "azurerm_virtual_wan" "test" {
+  name                = "acctest-vwan-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+}
+
+resource "azurerm_virtual_hub" "test" {
+  name                = "acctest-VHUB-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  virtual_wan_id      = azurerm_virtual_wan.test.id
+  address_prefix      = "10.0.2.0/24"
+}
+
+resource "azurerm_virtual_hub_connection" "test" {
+  name                      = "acctest-vhubconn-%d"
+  virtual_hub_id            = azurerm_virtual_hub.test.id
+  remote_virtual_network_id = azurerm_virtual_network.test.id
+}
+`, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger)
+}
+
+func testAccAzureRMVirtualHubRouteTable_basic(data acceptance.TestData) string {
+	template := testAccAzureRMVirtualHubRouteTable_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_virtual_hub_route_table" "test" {
+  name           = "acctest-routetable-%d"
+  virtual_hub_id = azurerm_virtual_hub.test.id
+  labels         = ["label1"]
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMVirtualHubRouteTable_requiresImport(data acceptance.TestData) string {
+	config := testAccAzureRMVirtualHubRouteTable_basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_virtual_hub_route_table" "import" {
+  name           = azurerm_virtual_hub_route_table.test.name
+  virtual_hub_id = azurerm_virtual_hub_route_table.test.virtual_hub_id
+  labels         = azurerm_virtual_hub_route_table.test.labels
+}
+`, config)
+}
+
+func testAccAzureRMVirtualHubRouteTable_complete(data acceptance.TestData) string {
+	template := testAccAzureRMVirtualHubRouteTable_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_virtual_hub_route_table" "test" {
+  name           = "acctest-routetable-%d"
```

Comment:
capitals where possible
```suggestion
  name           = "acctest-RouteTable-%d"
```

Reply:
updated